configure.ac

```diff
@@ -267,6 +267,8 @@ AS_IF([test "x$ax_cv_c_compiler_vendor" = "xgnu"], [
 		AS_CASE([${host_os}],
 			[mingw*|cygwin*], [
 				SHLIB_VERSION_ARG="-export-symbols \$(top_srcdir)/Win32/libsamplerate-0.def"],
+			[darwin*], [
+				SHLIB_VERSION_ARG="-Wl,-exported_symbols_list,\$(top_srcdir)/src/libsamplerate.exports"],
```

Comment:
in libsndfile it looks little different:

https://github.com/libsndfile/libsndfile/blob/06fd6f626e75bc57caa9ff43cfa913b66216cd66/configure.ac#L606

Reply:
That one has two separate `-Wl`s, mine has two combined into one. Same result.